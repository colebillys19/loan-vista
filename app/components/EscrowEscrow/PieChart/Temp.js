import React from 'react';
import { SvgIcon } from '@material-ui/core';

import { TempWrapper } from './styledComponents';

const Temp = () => (
  <TempWrapper>
    <SvgIcon style={{ height: '8.4rem', width: '8.4rem' }} viewBox="0 0 84 84">
      <circle fill="#ffffff" r="50%" transform="translate(42, 42)" />
    </SvgIcon>
  </TempWrapper>
);

export default Temp;
