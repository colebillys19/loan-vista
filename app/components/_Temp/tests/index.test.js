import React from 'react';
import { render } from 'react-testing-library';

import Temp from '../index';

describe('<Temp />', () => {
  it('Expect to not log errors in console', () => {
    const spy = jest.spyOn(global.console, 'error');
    render(<Temp />);
    expect(spy).not.toHaveBeenCalled();
  });

  it.skip('Should render and match the snapshot', () => {
    const {
      container: { firstChild },
    } = render(<Temp />);
    expect(firstChild).toMatchSnapshot();
  });
});
