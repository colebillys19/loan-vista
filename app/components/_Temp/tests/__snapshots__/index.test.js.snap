// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Temp /> Should render and match the snapshot 1`] = `<div />`;
